templates/files/conf/k8s-addons

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -eu
-# k8s 1.14.1
-KUBECTL_IMAGE=quay.io/giantswarm/docker-kubectl:1.14.1
+# k8s 1.16.4
+# WHEN UPDATING TAG DONT FORGET TO UPDATE VERSION ABOVE AS WELL
+KUBECTL_IMAGE={{.DockerRegistry}}/giantswarm/docker-kubectl:4377f52eeb22d766bbc59bdf71ce0724af61a4f6
```

Comment:
decided to go back to tags, as we don't have version tag created by default and this would mean someone has to do taht manually in china and that would probably get forgotten